crates/matrix-sdk-crypto/CHANGELOG.md

```diff
@@ -40,3 +40,6 @@
 - Fix handling of SAS verification start events once we have shown a QR code.
 
 - Fix a bug which could cause generated one-time-keys not to be persisted.
+
+- Fix parsing error for `POST /_matrix/client/v3/keys/signatures/upload`
+  responses generated by Synapse.
```

Comment:
```suggestion
  failure responses generated by Synapse.
```

Reply:
It's part of the `failures` key, but it's not actually the endpoint returning with an error so I'd prefer to not change it this way.